src/aiida/cmdline/commands/cmd_setup.py

```diff
@@ -137,6 +138,7 @@ def setup(
 
 
 @verdi.command('quicksetup')
+@decorators.deprecated_command('This command is deprecated, use `verdi presto --use-postgres` instead.')
```

Comment:
Is there now essentially a full feature parity between quicksetup and presto for its use case? I guess if more flexibility is required, the user should use `verdi profile setup`? Might be good to mention `verdi profile setup` in the deprecation message as well.

Reply:
In terms of the automated generation of PSQL user and database, I think so. But I have never really used `verdi quicksetup` myself so I cannot be a 100% sure. In any case, `verdi presto` (intentionally) does not support all the options that `verdi quicksetup` does. So there is a quite a bit of customization that is now no longer available. But for that users should resort to `verdi profile setup` anyway